front/src/Phaser/Entity/PlayerTexturesLoadingManager.ts

```diff
@@ -66,8 +69,12 @@ export const getRessourceDescriptor = (textureKey: string|BodyResourceDescriptio
 }
 
 const createLoadingPromise = (loadPlugin: LoaderPlugin, playerResourceDescriptor: BodyResourceDescriptionInterface) => {
-    return new Promise<void>((res, rej) => {
+    return new Promise<BodyResourceDescriptionInterface>((res) => {
+        const texture = loadPlugin.textureManager.get(playerResourceDescriptor.name);
+        if(texture.key !== '__MISSING'){
```

Comment:
you can simply use textureManager.exists() to check if the texture was already loaded.
Also remove the texture check  in lazyLoadPlayerCharacterTextures (line 37) since it is redundant with this one.